test/adapters/prebid.spec.js

```diff
@@ -823,6 +823,8 @@ describe('ADAPTER: Prebid', function() {
     describe('#fetchBids', function() {
         var activeSlots = null;
         var impressionID = null;
+        var partnerToEnableSRA = null;
```

Comment:
unit test case should be reading data from constants

Reply:
Removed